### PR TITLE
fix: add clean before tsc

### PR DESCRIPTION
**Pull request by xiaoluSF**

### CODE REVIEW

File: packages/neuron-wallet/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   "license": "MIT",
   "scripts": {
     "start": "cross-env electron .",
-    "start:dev": "tsc && cross-env electron .",
+    "start:dev": "yarn clean && tsc && cross-env electron .",
     "build": "tsc",
     "clean": "rm -rf dist/*",
```
Comment (Keith-CY) [inline packages/neuron-wallet/package.json]: does `rm -rf` works on windows?

Comment (xiaoluSF) [reply packages/neuron-wallet/package.json]: now we just think about macOS and Linux.

Comment (classicalliu) [reply packages/neuron-wallet/package.json]: Only **package** on macOS and linux, support development on windows

Comment (xiaoluSF) [reply packages/neuron-wallet/package.json]: 👌, I try it.

**Pull request closed without merging**